Python/cryptocurrency_converter/README.md

```diff
@@ -0,0 +1,10 @@
+##Crypto Currency Converter
+ 
+This is a cryptocurrency converter using coinbase api
+
+* __Coinbase API__
+    
+    *We have used coinbase API to get exchange rates.
+    *getting exchange rates through coinbase API is free and doesn't requires an API key.
```

Comment:
Left-Align them. Take a look at other READMEs to find how to have a good Doc for your Script.